cc_core.c

```diff
@@ -3000,13 +3000,31 @@ void global_assignment(void)
 		globals_list = emit("_contents\n", globals_list);
 		globals_list = emit(parse_string(global_token->s), globals_list);
 	}
-	else
-	{
-		line_error();
-		fputs("Received ", stderr);
-		fputs(global_token->s, stderr);
-		fputs(" in program\n", stderr);
-		exit(EXIT_FAILURE);
+	else {
+		struct token_list* symbol = sym_lookup(global_token->s, global_constant_list);
+		if(NULL != symbol)
+		{
+			globals_list = emit("%", globals_list);
+			globals_list = emit(symbol->arguments->s, globals_list);
+
+			/* broken for big endian architectures */
```

Comment:
Is there some difficulty to fixing this?

Reply:
It's not due to big endian architectures. See https://github.com/oriansj/M2-Planet/issues/63 where I root caused this issue.